stratum/hal/lib/tdi/tofino/tofino_chassis_manager_test.cc

```diff
@@ -420,9 +420,9 @@ TEST_F(TofinoChassisManagerTest, ApplyDeflectOnDrop) {
   ASSERT_OK(PushBaseChassisConfig(&builder));
 
   EXPECT_CALL(*port_manager_,
-              SetDeflectOnDropDestination(device, kPortId + kSdkPortOffset, 4))
+              SetDeflectOnDropDestination(kDevice, kPortId + kSdkPortOffset, 4))
       .Times(AtLeast(1));
-  EXPECT_CALL(*port_manager_, SetDeflectOnDropDestination(device, 56789, 1))
+  EXPECT_CALL(*port_manager_, SetDeflectOnDropDestination(kDevice, 56789, 1))
```

Comment:
PR looks good to me. I see lot of numbers floating around, can we change them to relevant macros.

Reply:
For this test case, probably not. The numeric value needs to match the value in the protobuf, which is specified as a JSON strong:

```c
TEST_F(TofinoChassisManagerTest, ApplyDeflectOnDrop) {
  const std::string kVendorConfigText = R"PROTO(
    tofino_config {
      node_id_to_deflect_on_drop_configs {
        key: 7654321
        value {
          drop_targets {
            port: 12345
            queue: 4
          }
          drop_targets {
            sdk_port: 56789
            queue: 1
          }
        }
      }
    }
  )PROTO";

  VendorConfig vendor_config;
  ASSERT_OK(ParseProtoFromString(kVendorConfigText, &vendor_config));

  ChassisConfigBuilder builder;
  builder.SetVendorConfig(vendor_config);
  ASSERT_OK(PushBaseChassisConfig(&builder));

  EXPECT_CALL(*port_manager_,
              SetDeflectOnDropDestination(kDevice, kPortId + kSdkPortOffset, 4))
      .Times(AtLeast(1));
  EXPECT_CALL(*port_manager_, SetDeflectOnDropDestination(kDevice, 56789, 1))
      .Times(AtLeast(1));
```
If we make the port number symbolic, the reader would have to look in _three_ places (the JSON, the constant symbol, and the call to SetDeflectOnDropDestination) to put the pieces together. This way, they only have to look in two places.